openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -236,7 +239,11 @@ def test_failure_with_valid_symmetric_jwt(self):
         assert '_auth_user_id' not in self.client.session
         assert response.status_code == 401
 
-    def test_failure_with_valid_asymmetric_jwt_and_disabled_user(self):
+    @mock.patch.object(NotJwtRestrictedApplication, 'has_permission')
+    @mock.patch.object(JwtAuthentication, 'authenticate')
+    def test_failure_with_valid_asymmetric_jwt_and_disabled_user(self, jwt_authenticate, jwt_has_permissions):
```

Comment:
1. Can we add a test case where disabled user is filtered by auth class?
2. This function name describes the above case, can we rename it to to show the intent.

Reply:
@jawad-khan Ideally those test cases are supposed to be where the JwtAuthentication class exists, and those test cases are already added as [we can see here](https://github.com/openedx/edx-drf-extensions/blob/d483e6651b584ff787bc9678dc5237264145c281/edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py#L199).

Reply:
@jawad-khan For renaming, what do you suggest we rename it to?